generator/Makefile

```
@@ -155,6 +158,15 @@ $(MIBDIR)/KEEPALIVED-MIB:
 	@echo ">> Downloading KEEPALIVED-MIB"
 	@curl $(CURL_OPTS) -o $(MIBDIR)/KEEPALIVED-MIB $(KEEPALIVED_URL)
 
+$(MIBDIR)/.kemp-lm:
+	$(eval TMP := $(shell mktemp))
+	@echo ">> Downloading Kemp LM MIBs to $(TMP)"
+	@curl $(CURL_OPTS) -o $(TMP) $(KEMP_LM_URL)
+	@unzip -j -d $(MIBDIR) $(TMP) *.txt
+	@sed -i -E "s/([0-9]{12})[0-9]{2}Z/\1Z/" $(MIBDIR)/*.RELEASE-B100-MIB.txt
```

Would you mind adding a comment here that this is a workaround? Plus a minor suggestion for the regexp.

```suggestion
	# Workaround invalid timestamps.
	@sed -i -E 's/"([0-9]{12})[0-9]{2}Z"/"\1Z"/' $(MIBDIR)/*.RELEASE-B100-MIB.txt
```

---

I agree on your suggestions and have updated the commit.
Initially I had a similar comment included locally, but was annoyed/confused of the fact that `make mibs` printed the comment, thus wasn't confident enough to include it.